lib/bcdice/game_system/DoubleCross.rb

```diff
@@ -286,6 +293,18 @@ def roll_emotion_table
         return Result.new(output_parts.join(' ＞ '))
       end
 
+      # ハプニングチャートを振る
+      #
+      #
+      #
+      # @return [Result]
+      def roll_happening_chart
+        result = self.class::HAPPENING_CHART.roll(@randomizer)
+        name = translate("DoubleCross.HC.name")
+        output_parts = ["#{name}(#{result.sum})", result.content]
+        return Result.new(output_parts.join(' ＞ '))
+      end
+
```

Comment:
```markdown
```suggestion

      # RW Prologue Chartを振る
       # @return [Result]
      def roll_prologue_chart_positive
        result = self.class::PROLOGUE_CHART_POSITIVE.roll(@randomizer)
        name = translate("DoubleCross.PCP.name")
        output_parts = ["#{name}(#{result.sum})", result.content]
        return Result.new(output_parts.join(' ＞ '))
      end
      # RW Prologue Chartを振る 2
       # @return [Result]
      def roll_prologue_chart_negative
        result = self.class::PROLOGUE_CHART_NEGATIVE.roll(@randomizer)
        name = translate("DoubleCross.PCN.name")
        output_parts = ["#{name}(#{result.sum})", result.content]
        return Result.new(output_parts.join(' ＞ '))
      end